src/Maths/Silk.Net.Maths.Tests/Silk.Net.Maths.Tests.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.0;netcoreapp5.0</TargetFrameworks>
```

Comment:
`net5.0` here too. If you are experiencing issues with the latest preview (preview 5, use `dotnet --version` to check your current version) let me know.